<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="attributesButton">
         <property name="toolTip">
          <string>View IMAQdx attributes</string>
         </property>
         <property name="text">
          <string>Attributes</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/qtutils/fugue/table-export.png</normaloff>:/qtutils/fugue/table-export.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="snapButton">
         <property name="toolTip">
          <string>Take a single image</string>
         </property>
         <property name="text">
          <string>Snap</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/qtutils/fugue/control-stop.png</normaloff>:/qtutils/fugue/control-stop.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="continuousButton">
         <property name="toolTip">
          <string>Acquire images continuously</string>
         </property>
         <property name="text">
          <string>Continuous</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/qtutils/fugue/control.png</normaloff>:/qtutils/fugue/control.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="toolTip">
          <string>Stop acquiring images</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/qtutils/fugue/control-stop-square.png</normaloff>:/qtutils/fugue/control-stop-square.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QDoubleSpinBox" name="maxRateSpinBox">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum frame rate in continuous acquisition mode. Use this to reduce load on the computer during continuous acquisition,otherwise frames are acquired as quickly as possible, which may not be desirable on slower computers.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="specialValueText">
            <string>No max rate</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="suffix">
            <string> fps</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>9999.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="noMaxButton">
           <property name="toolTip">
            <string>No max rate</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/qtutils/fugue/arrow-turn-180-left.png</normaloff>:/qtutils/fugue/arrow-turn-180-left.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="fpsLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../clones/qtutils/qtutils/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
